index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Online Library</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <h1 class="title">Online Library</h1>
+        <form action="#" class="book-form">
+            <input type="text" class="book-form__input book-form__input--id" placeholder="ID" name="id" required>

```

Comment:
I can't find style for `book-form__input--id` . Do you know if it is needed at all?

Reply:
> I can't find style for `book-form__input--id` . Do you know if it is needed at all?

there's no style, but I made it, because I needed to get each element for variables `formId`, `formTitle`, etc... But I also created global `book-form__input` to style them all. Maybe there's a way to get each element without separate class names?